box-share-sdk/src/main/java/com/box/androidsdk/share/usx/fragments/InviteCollaboratorsFragment.java

```diff
@@ -157,18 +159,18 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
             //need to log Exception
             BoxLogUtils.e(InviteCollaboratorsFragment.class.getName(), "Fetch roles request failed",
                     presenter.getException());
-            showToast(getString(presenter.getStrCode())); //was just collaborationfragment
+            showToast(getString(presenter.getStrCode()));
         }
     };
 
     private Observer<PresenterData<BoxIteratorInvitees>> onInviteesChanged = presenter -> {
```

Comment:
Could you rename `presenter` to `presenterData` or `uiData` or something similar. Presenter in Android has a different meaning, I don't want the reader to get confused. If you are interested to know more, read about MVP (Model-View-Presenter) pattern.